USE-CASES/govtech-health-monitoring.md

```diff
@@ -0,0 +1,117 @@
+## Title: Public Health Monitoring
+
+### Submitter(s): 
+
+Jennifer Lin
+
+### Reviewer(s):
+
+Michael McCool, Michael Lagally
+
+### Tracker Issue ID:
+
+<please leave blank>
+
+### Category:
+
+<please leave blank>
+
+### Class: 
+
+<please leave blank>
+
+### Status: 
+
+<please leave blank>
+
+### Target Users
+
+Agencies, companies and other organizations in a Smart City with 
+significant foot traffic.
+
+### Motivation:
+
+A system to monitor the health of people in public places is useful to
+control the spread of infectious diseases.  In particular, we would like
+to identify individuals with temperatures outside the norm (i.e. running
+a fever) and then take appropriate action.
+
+This mechanism should be non-invasive and non-contact since the solution
+should not itself contribute to the spread of infectious diseases.
+
+### Expected Devices:
+
+One of the following:
+* A thermal camera.
+* Face detection (AI) service.
+
```

Comment:
cloud service for central data aggregation and analytics.

Reply:
added, as optional (since aggregation is optional).  Note there is some additional text and discussion around this now.  There is also a new requirement: identifying/distinguishing unique individuals to avoid double-counting.